### PR TITLE
Make it easier to copy the instruction in the issue template

### PR DESCRIPTION
**Pull request by toslunar**
A closing backtick has been missing.

### CODE REVIEW

**Comment by crcrpar** [comment]
related to https://github.com/chainer/chainer/pull/6165#issuecomment-460114573.

+1 for this because even if markdown is not supoorted, backticks would make it easier to read the runtime information.

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -7,7 +7,12 @@ please take a look at [our contribution guide](https://docs.chainer.org/en/stabl
 Specifically, if it is a bug report, these information are very helpful:
 
 * Conditions
-<!-- If you're using Chainer 4.0+, you can also get this information by typing `python -c 'import chainer; chainer.print_runtime_info()'. -->
+<!--
+If you're using Chainer 4.0+, you can also get this information by typing the following:
```
Comment (kmaehashi) [inline .github/ISSUE_TEMPLATE.md]: Let's remove `If you're using Chainer 4.0+, ` part, as we no longer support Chainer v3.

**Comment by Crissman** [comment]
LGTM. Jenkins, test this please.

**Comment by Crissman** [comment]
LGTM. Merging.

**Pull request merged**